### PR TITLE
Merge Hotfix after release

### PR DESCRIPTION
**Pull request by mwithi**

### CODE REVIEW

File: src/main/java/org/isf/dicom/service/DicomIoOperations.java [hunk 1]
```diff
@@ -168,8 +169,12 @@ public boolean isCodePresent(Long code) throws OHServiceException {
 	 * @return <code>true</code> if the code is already in use, <code>false</code> otherwise
 	 * @throws OHServiceException
 	 */
-	public boolean isSeriePresent(String dicomSeriesNumber) throws OHServiceException {
-		List<FileDicom> result = repository.findAllWhereDicomSeriesNumber(dicomSeriesNumber);
-		return !result.isEmpty();
+	public boolean isSeriesPresent(String dicomSeriesNumber) throws OHServiceException
+	{
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/dicom/service/DicomIoOperations.java:173]: Don't the braces usually go on the same line?

**Comment by dbmalkovsky** [comment]
Other than a few formatting issues with braces; LGTM

File: src/main/java/org/isf/dicom/service/DicomIoOperations.java (hunk 1 above)
Comment (mwithi) [reply src/main/java/org/isf/dicom/service/DicomIoOperations.java]: yep! done. Thank you!

**Comment by mwithi** [comment]
> Other than a few formatting issues with braces; LGTM

Thanks!

> Please double check the version number in the POM to make sure it is what you want; thanks.

Yes, let's align all of them to the same *real* common version, then we see how to proceed. Thanks!

**Pull request merged**